src/ui/pages/codi/zcl_abapgit_gui_page_whereused.clas.abap

```diff
@@ -0,0 +1,222 @@
+CLASS zcl_abapgit_gui_page_whereused DEFINITION
+  PUBLIC
+  FINAL
+  INHERITING FROM zcl_abapgit_gui_component
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+    INTERFACES:
+      zif_abapgit_gui_page_title,
+      zif_abapgit_gui_event_handler,
+      zif_abapgit_gui_hotkeys,
+      zif_abapgit_gui_menu_provider,
+      zif_abapgit_gui_renderable,
+      zif_abapgit_html_table.
+
+    CLASS-METHODS create
+      IMPORTING
+        iv_package     TYPE devclass
+      RETURNING
+        VALUE(ri_page) TYPE REF TO zif_abapgit_gui_renderable
+      RAISING
+        zcx_abapgit_exception.
+
+    METHODS constructor
+      IMPORTING
+        iv_package TYPE devclass
+      RAISING
+        zcx_abapgit_exception.
+
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+    CONSTANTS:
+      BEGIN OF c_action,
+        refresh TYPE string VALUE 'refresh',
+      END OF c_action.
+
+    CONSTANTS c_title TYPE string VALUE 'Where Used'.
+    DATA mv_package TYPE devclass.
+    DATA mi_table TYPE REF TO zcl_abapgit_html_table.
+
+    METHODS init_table_component
+      RAISING
+        zcx_abapgit_exception.
+
+ENDCLASS.
+
+
+
+CLASS ZCL_ABAPGIT_GUI_PAGE_WHEREUSED IMPLEMENTATION.
+
+
+  METHOD constructor.
+    super->constructor( ).
+    mv_package = iv_package.
+
+    IF zcl_abapgit_factory=>get_sap_package( iv_package )->exists( ) = abap_false.
+      zcx_abapgit_exception=>raise( |Package { iv_package } does not exist| ).
+    ENDIF.
+
+  ENDMETHOD.
+
+
+  METHOD create.
+
+    DATA lo_component TYPE REF TO zcl_abapgit_gui_page_whereused.
+
+    CREATE OBJECT lo_component
+      EXPORTING
+        iv_package = iv_package.
+
+    ri_page = zcl_abapgit_gui_page_hoc=>create( lo_component ).
+
+  ENDMETHOD.
+
+
+  METHOD init_table_component.
+
+    DATA ls_sorting_state TYPE zif_abapgit_html_table=>ty_sorting_state.
+
+    IF mi_table IS BOUND.
+      RETURN.
+    ENDIF.
+
+    IF ls_sorting_state-column_id IS INITIAL.
+      ls_sorting_state-column_id = 'package'.
+    ENDIF.
+
+    mi_table = zcl_abapgit_html_table=>create(
+      ii_renderer              = me
+      is_initial_sorting_state = ls_sorting_state ).
+    mi_table->define_column_group(
+        iv_group_title = 'Repo object'
+        iv_group_id    = '' " No need
+      )->define_column(
+        iv_column_id    = 'dep_package'
+        iv_column_title = 'Pkg'
```

Comment:
No abbreviations necessary... "Package", "Type", "Name" (same as in [repo view](https://github.com/abapGit/abapGit/blob/ecdd17162a2c5e14d4feae99ef2f6f9b157db7b7/src/ui/pages/zcl_abapgit_gui_page_repo_view.clas.abap#L1040))

Reply:
done